[2022-06-27 13:32:05,870] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.accurate scheduled__2022-03-22T00:00:00+00:00 [queued]>
[2022-06-27 13:32:05,901] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.accurate scheduled__2022-03-22T00:00:00+00:00 [queued]>
[2022-06-27 13:32:05,903] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:05,904] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:05,905] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:05,936] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): accurate> on 2022-03-22 00:00:00+00:00
[2022-06-27 13:32:05,944] {standard_task_runner.py:52} INFO - Started process 57645 to run task
[2022-06-27 13:32:05,979] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'accurate', 'scheduled__2022-03-22T00:00:00+00:00', '--job-id', '460', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmplsyn6ojm', '--error-file', '/tmp/tmp8kfr9l_z']
[2022-06-27 13:32:05,980] {standard_task_runner.py:80} INFO - Job 460: Subtask accurate
[2022-06-27 13:32:06,078] {task_command.py:370} INFO - Running <TaskInstance: dag1.accurate scheduled__2022-03-22T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:06,271] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-03-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-22T00:00:00+00:00
[2022-06-27 13:32:06,273] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 13:32:06,275] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-06-27 13:32:06,321] {subprocess.py:85} INFO - Output:
[2022-06-27 13:32:06,329] {subprocess.py:92} INFO - accurate
[2022-06-27 13:32:06,335] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-27 13:32:06,391] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=accurate, execution_date=20220322T000000, start_date=20220627T133205, end_date=20220627T133206
[2022-06-27 13:32:06,495] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:06,555] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
